tests/execution/apollo-federation-entities.md

```diff
@@ -21,7 +21,7 @@ type User @call(steps: [{query: "user", args: {id: "{{.value.id}}"}}]) {
 ```graphql @file:posts.graphql
 type Post @expr(body: {id: "{{.value.id}}", title: "post-title-{{.value.id}}"}) {
   id: Int!
-  title: String!
+  title: String! @override(from: "name")
```

Comment:
Lets add a separate test to support extra directives. This feature is independent of federation.

Reply:
done